test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPushCommandTest.cs

```diff
@@ -336,13 +338,6 @@ public void PushCommand_PushToServerBasicAuth()
                     using (var server = new MockServer())
                     {
                         server.Listener.AuthenticationSchemes = AuthenticationSchemes.Basic;
-                        server.Get.Add("/nuget", r =>
```

Comment:
And i can log a tracking bug. Based on our conversation last Friday, we do not need to worry about this for now.